<?xml version="1.0" ?>
<launch>
  <!--
      Launch file for the admittance controller
  -->

  <arg name="TOPIC_ARM_STATE" default="/cartesian_velocity_controller_sim/ee_state"/>
  <arg name="TOPIC_ARM_COMMAND" default="/cartesian_velocity_controller_sim/command_cart_vel"/>
  <arg name="TOPIC_WRENCH_STATE" default="/wrench"/>
  <arg name="TOPIC_CART_POSE_CMD" default="/desired_carteisan_pose_cmd"/>


  <param name="topic_arm_state" value="$(arg TOPIC_ARM_STATE)"/>
  <param name="topic_arm_command" value="$(arg TOPIC_ARM_COMMAND)"/>
  <param name="topic_wrench_state" value="$(arg TOPIC_WRENCH_STATE)"/>
  <param name="topic_cart_pose_cmd" value="$(arg TOPIC_CART_POSE_CMD)"/>



  <rosparam file="$(find hybrid_position_force_control)/config/AdmittanceParams.yaml" command="load"/>

  <node pkg="hybrid_position_force_control" name="hybrid_position_force_control" type="hybrid_position_force_control_node" output="screen"/>

</launch>
